ci/test-install/storage_install_test.cc

```diff
@@ -0,0 +1,55 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/storage/client.h"
```

Comment:
Wouldn't this be `#include <google/cloud/storage/client.h>` when installed? If not also trying to understand why that's not that case.

Reply:
Quotes or brackets depends on what the style guide that the application developer adopts.

With CMake you can use either, it adds the `-I` flag and that affects the search path for brackets and quotes.

With Bazel the default is to use the `-iquote` flag, which requires quotes:

https://stackoverflow.com/questions/46498441/how-to-allow-angle-brackets-when-including-a-bazel-header

I decided to leave the test with quotes because it was easy to do, more natural for the folks using Bazel, and sort of "meh" for everybody else.

Reply:
Sg, thanks. I think that's fair for now.